File: General Program/2-stringReverse.js
// write a function to reverse a string

function reverse(str){
    // add whatever parameters you deem necessary - good luck!
    let word = str.split('');
    let i = 0;
    let j = word.length;
    while(i<j){
        let temp = word[i];
        word[i] = word[j];
        word[j] = temp;
        i++;
        j--;
    }
    return word.join("");
  }
  
  let rev = reverse('string');
  console.log(rev);
  
  // reverse('awesome') // 'emosewa'
  // reverse('rithmschool') // 'loohcsmhtir'

File: python/udemy/general Programs/2. String Reverse.py
# write a function to reverse a string

def reverse(str):
    listWord = list(str)
    start = 0
    end = len(listWord) - 1
    while start <= end:
        temp = listWord[start]
        listWord[start] = listWord[end]
        listWord[end] = temp

        start += 1
        end -= 1
    return ''.join(listWord)

print(reverse("hello"))
print(reverse("sumant shah"))